frontend/containers/SignIn/SignIn.jsx

```diff
@@ -91,24 +91,22 @@ class SignIn extends Component {
     const {email, password, userHasInteracted} = this.state
     const hasConnectionIssues = state.app.networkStatus !== Constants.NETWORK_OK
     const {
-      whitelabel: {logo, poweredBy, backgroundImage}
-    } = state.app.config || {
-      whitelabel: {logo: '', poweredBy: false, backgroundImage: ''}
-    }
+      whitelabel: {enabled, path, logo, poweredBy, backgroundImage}
```

Comment:
I know this has transitioned from a previous PR, but I'm really not a fan of this nested object destructuring syntax. Not only it looks confusing to me, but we end up using non-descriptive variable names for no reason. For example, the variable `enabled` is supposed to represent whether *the whitelabel functionality* has been enabled.

Are you happy for me to change that as part of this PR?

Reply:
Yeah sure, that was confusing for me too, but didn’t want to mess with the status quo!